doc/3/controllers/auth/validate-my-credentials/index.md

```diff
@@ -0,0 +1,35 @@
+---
+code: true
+type: page
+title: ValidateMyCredentials
+description: Validate the current user's credentials for the specified strategy.
+---
+
+# ValidateMyCredentials
+
+Validate the current user's credentials for the specified strategy. The `result` field is `true` if the provided credentials are valid; otherwise an error is triggered. This route does not actually create or modify the user credentials. The credentials to send will depend on the authentication plugin and authentication strategy.
```

Comment:
```suggestion
Validates the current user's credentials for the specified strategy. The `result` field is `true` if the provided credentials are valid; otherwise an error is triggered. This route does not actually create or modify the user credentials. The credentials to send will depend on the authentication plugin and authentication strategy.
```